### DISCUSSION TITLE
How to get e-invoice PDF Url

### DISCUSSION THREAD

**Discussion by kindacoder** [fattureincloud/api#390] [Q&A] [answered]
Hey Folks,

I successfully retrieved the URL of the invoice in the "issueDocument" API response, but I'm facing difficulty obtaining the URL for the e-invoice PDF. Could someone please advise on how to access this information?

Thank you.

**Comment by valmoz** [comment]
Hi @kindacoder to be honest I don't know what are you trying to obtain.. It should be the same link for both invoices and e-invoices

**Reply by kindacoder** [reply]
Actually, I am talking about these invoices and e-invoices.


Let me know.


### Invoice which received from URL:


[Fattura-25_fatt_2023-Alessandra-Ojetti (1).pdf](https://github.com/fattureincloud/api/files/13815617/Fattura-25_fatt_2023-Alessandra-Ojetti.1.pdf)





### E-invoice from console:


![image_anno](https://github.com/fattureincloud/api/assets/28186433/31921c16-1c02-4270-96cc-6aa33e7af6a1)
 

How to get the second one?

**Reply by emajo** [reply]
Hi @kindacoder, which functionality of the web application did you use to obtain the second pdf? Perhaps you downloaded the e-invoice xml and you are opening it with a different reader, because it does not seem a pdf generated by Fatture in Cloud.

Thanks,
Emanuele.

**Reply by kindacoder** [reply]
Yes, correct. I am previewing the e-invoice xml @emajo

**Reply by kindacoder** [reply]
is there any way to get that into pdf form? @emajo

**Reply by emajo** [reply, marked as answer]
You can download the xml using the [get einvoice xml](https://developers.fattureincloud.it/api-reference#get-/c/-company_id-/issued_documents/-document_id-/e_invoice/xml) api call, but if you want a pdf as result the only one we offer is the one you already found using the get issued document method, which is the same for an electronic or not electronic invoice.

**Reply by kindacoder** [reply]
got it. thanks
